examples/category-results-materialize.html

```diff
@@ -18,18 +18,23 @@
           ng-model="demo.model"
           placeholder="Enter a car make (e.g. Ford)"
           autofocus="true"
+          is-selectable="demo.isSelectable(suggestion)"
+          multiple="true"
           source="demo.sourceFn(query)">
 
         <ngc-omnibox-suggestions class="collection with-header">
           <dl ngc-omnibox-suggestion-category>
             <dt class="collection-header">
-              <h5>{{suggestion.make_country}}</h5>
+              <h5 ngc-omnibox-suggestion-header class="collection-item"
+                  ng-class="{'active': omnibox.isHighlighted(suggestion)}">
```

Comment:
This is the class used by the materialize css library, not ours. We only add aria stuff, never any css classes.